src/applications/disability-benefits/4142/config/form.js

```diff
@@ -0,0 +1,273 @@
+
+import dateRangeUI from 'us-forms-system/lib/js/definitions/dateRange';
+import environment from '../../../../platform/utilities/environment';
+
+import IntroductionPage from '../containers/IntroductionPage';
+import ConfirmationPage from '../containers/ConfirmationPage';
+
+// import {
+//   uiSchema as facilityAddressUiSchema,
+//   schema as facilityAddressSchema
+// } from '../pages/facilityAddress';
+
+// const { } = fullSchema4142.properties;
+
+const {
+  fullName,
+  ssn,
+  date,
+  address,
+  phone,
+  dateRange
+} = fullSchema4142.definitions;
+
+import {
+  recordHelp,
+  recordReleaseDescription,
+  aboutPrivateMedicalRecords,
+  limitedConsentDescription,
+  recordReleaseSummary,
+  disabilityNameTitle,
+  validateZIP,
+  transform,
+} from '../helpers';
+
+import { validateDate } from 'us-forms-system/lib/js/validation';
+
+// Define all the form pages to help ensure uniqueness across all form chapters
+const formPages = {
+  uploadInformation: 'uploadInformation',
+  treatmentHistory: 'treatmentHistory',
+  recordReleaseSummary: 'recordReleaseSummary',
+};
+
+const formConfig = {
+  urlPrefix: '/',
+  submitUrl: `${environment.API_URL}/v0/private_medical_records/submit`,
+  trackingPrefix: '21-4142-',
+  introduction: IntroductionPage,
+  confirmation: ConfirmationPage,
+  formId: '21-4142',
+  version: 0,
+  prefillEnabled: true,
+  savedFormMessages: {
+    notFound: 'Please start over to apply for benefits.',
+    noAuth: 'Please sign in again to continue your application for benefits.',
+  },
+  transformForSubmit: transform,
+  title: '4142 Private Medical Record Release Form', // TODO: Verify the title and subtitle
+  defaultDefinitions: {
+    fullName,
+    ssn,
+    date,
+    dateRange,
+    phone,
+    address
+  },
+  chapters: {
+    applicantInformation: {
+      title: 'Apply for disability increase', // TODO: Verify title
+      pages: {
+        [formPages.uploadInformation]: {
+          // THIS IS NOT A REAL PAGE; WILL BE THROWN OUT IN 526 INTEGRATION TODO
+          path: 'private-medical-record-upload',
+          title: 'Supporting Evidence',
+          uiSchema: {
+            'ui:title': disabilityNameTitle,
+            'view:uploadRecords': {
+              'ui:description': aboutPrivateMedicalRecords,
+              'ui:title': aboutPrivateMedicalRecords,
+              'ui:widget': 'radio',
+              'ui:options': {
+                // TODO depends: (formData) => !formData.view:uploadRecs
+                labels: {
+                  yes: 'Yes',
+                  no: 'No, please get them from my doctor.',
+                },
+              },
+            },
+            'view:privateRecordsChoiceHelp': {
+              'ui:description': recordHelp,
+            },
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              'view:uploadRecords': {
```

Comment:
You probably want a boolean type here, for which you can specify the label names if you use the `yesNo` widget e.g.:
```
'ui:widget': 'yesNo',
'ui:options: {
  labels: {
    Y: 'Yes, I have uploaded all my supporting documents.',
    N: 'No, I have some extra information that I will submit to VA later.'
  }
}